src/NHibernate/Cfg/Environment.cs

```diff
@@ -258,6 +258,17 @@ public static string Version
 		/// </remarks>
 		public const string OracleUseNPrefixedTypesForUnicode = "oracle.use_n_prefixed_types_for_unicode";
 
+		/// <summary>
+		/// <para>Set whether tracking the session id or not. When <see langword="true"/>, each session 
+		/// will have an unique <see cref="Guid"/> that can be retrieved by <see cref="ISessionImplementor.SessionId"/>,
+		/// otherwise <see cref="ISessionImplementor.SessionId"/> will always be <see cref="Guid.Empty"/>.</para>
```

Comment:
I would add something like "Session id is used for logging purpose."

Reply:
[Added](https://github.com/nhibernate/nhibernate-core/pull/1394/commits/8ca4ba8e4a32a800bf2a48de9b371387e4f034ae#diff-2063778a671420f6e1bceee0dd202b1bR264).